mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -681,18 +686,22 @@
     </string>
 
     <!-- Tip displayed on home view explaining how to request the desktop version of a website.
-     Argument 1 will be replaced with a line break in the application. -->
-    <string name="tip_request_desktop"><![CDATA[Rather see the full desktop site?%1$s Menu > Request desktop site]]></string>
+      %1$s is a placeholder for the app name. -->
+    <string name="tip_request_desktop2">Want to see the full version of a site? Switch on Desktop Site from the %1$s menu&gt;</string>
```

Comment:
@mcarare Just wondering if `menu&gt;` is expected in this string? Should this rather be just "menu"?

Reply:
THanks for getting this. It was an extra character:  > I have removed it in https://github.com/mozilla-mobile/focus-android/pull/5151/files.